$MirOS: ports/multimedia/mplayer/patches/patch-libmpdemux_aviheader_c,v 1.1.7.1 2005/03/18 15:50:49 tg Exp $
--- libmpdemux/aviheader.c.orig	Tue Feb 22 20:23:56 2005
+++ libmpdemux/aviheader.c	Thu May 19 20:32:06 2005
@@ -411,7 +411,7 @@ while(1){
   }
   mp_msg(MSGT_HEADER,MSGL_DBG2,"list_end=0x%X  pos=0x%X  chunksize=0x%X  next=0x%X\n",
       (int)list_end, (int)stream_tell(demuxer->stream),
-      chunksize, (int)chunksize+stream_tell(demuxer->stream));
+      (int)chunksize, (int)chunksize+stream_tell(demuxer->stream));
   if(list_end>0 &&
      chunksize+stream_tell(demuxer->stream) == list_end) list_end=0;
   if(list_end>0 && chunksize+stream_tell(demuxer->stream)>list_end){
